examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,215 @@
+
+---
+
+## How to Deploy
+
+### Prerequisites
+- An existing Amazon Managed Grafana (AMG) Workspace.
+  - As of this writing (February 3, 2022), the AWS Terraform Provider does not support Amazon Managed Grafana, so it must be manually created beforehand. Instructions [here](https://docs.aws.amazon.com/grafana/latest/userguide/getting-started-with-AMG.html).
+
+#### Generate a Grafana API Key
+- Give admin access to the SSO user you set up when creating the AMG Workspace:
+  - In the AWS Console, navigate to Amazon Grafana. In the left navigation bar, click __All workspaces__, then click on the workspace name you are using for this example.
+  - Under __Authentication__ within __AWS Single Sign-On (SSO)__, click __Configure users and user groups__
+  - Check the box next to the SSO user you created and click __Make admin__
+- Navigate back to the Grafana Dashboard. If you don't see the gear icon in the left navigation bar, log out and log back in.
+- Click on the gear icon, then click on the __API keys__ tab.
+- Click __Add API key__, fill in the _Key name_ field and select _Admin_ as the Role.
+- Copy your API key into `dev.tfvars` under `grafana_api_key` 
+
+### Deployment Steps
+- Clone this repository: 
+```
+git clone https://github.com/aws-samples/aws-eks-accelerator-for-terraform.git
+```
+- Initialize a working directory
+```
+cd examples/observability/eks-cluster-with-observability
+terraform init
+```
+- Fill-in the values for the variables in `dev.tfvars`
+  - The password for OpenSearch must be a minimum of eight characters with at least one uppercase, one lowercase, one digit, and one special character.
+- Verify the resources created by this execution:
+```
+export AWS_REGION=<ENTER YOUR REGION>   # Select your own region
+terraform validate
+terraform plan -tf-vars=dev.tfvars
+```
+- Deploy resources:
+ ```
+ terraform apply -tf-vars=dev.tfvars
+ ```
+- Add the cluster to your kubeconfig: 
+```
+aws eks --region $AWS_REGION update-kubeconfig --name aws001-preprod-observability-eks
+```
+
+`terraform apply` will provision a new EKS cluster with Fluent Bit, Prometheus, and a sample workload. It will also provision Amazon Managed Prometheus to ingest metrics from Prometheus, an Amazon OpenSearch service domain for ingesting logs from Fluent Bit, and an EC2 instance so we can test OpenSearch.
+
+#### Verify that the Resources Deployed Sucessfully
+
+- Check that the EC2 Instance we use to test OpenSearch is running in the EC2 Console.
+
+- Check that the status of OpenSearch is green:
+Navigate to Amazon OpenSearch in the AWS Console and select the __opensearch__ domain. Verify that *Cluster Health* under *General Information* lists Green.
+
+- Check that Amazon Managed Prometheus is healthy:
```

Comment:
How do you check it's healthy for Amazon Managed Prometheus? Did you men to say verify whether Amazon Managed Prometheus workspace is created successfully